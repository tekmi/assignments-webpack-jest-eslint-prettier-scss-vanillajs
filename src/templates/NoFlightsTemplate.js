export default search => `
  <li class="rw-autosuggest__results-flights">
      <ul class="rw-autosuggest-group">
          <li class="rw-autosuggest__results-item--noflight" data-ft-element="flights-search-results-notfound">
              Unfortunately, we cannot find a flight with '${search}'. Please, try again.
          </li>
      </ul>
  </li>
`;
