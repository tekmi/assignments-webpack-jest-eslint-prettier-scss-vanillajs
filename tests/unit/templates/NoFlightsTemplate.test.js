import NoFlightsTemplate from "../../../src/templates/NoFlightsTemplate";

describe("NoFlightsTemplate", () => {
  it("Renders properly with searched phrase", () => {
    expect(NoFlightsTemplate("San Francisco")).toMatchSnapshot();
  });
});
