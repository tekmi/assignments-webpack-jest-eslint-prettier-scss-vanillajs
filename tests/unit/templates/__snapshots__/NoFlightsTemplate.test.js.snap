// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoFlightsTemplate Renders properly with searched phrase 1`] = `
"
  <li class=\\"rw-autosuggest__results-flights\\">
      <ul class=\\"rw-autosuggest-group\\">
          <li class=\\"rw-autosuggest__results-item--noflight\\">
              Unfortunately, we cannot find a flight with 'San Francisco'. Please, try again.
          </li>
      </ul>
  </li>
"
`;
